datanode/snapshot/service_create_snapshot.go

```diff
@@ -89,7 +92,7 @@ func (b *Service) createSnapshot(ctx context.Context, chainID string, fromHeight
 			err = b.snapshotData(ctx, copyDataTx, dbMetaData, currentSnapshot, historySnapshot, chainID, toHeight)
 			b.snapshotInProgress.Store(false)
 			if err != nil {
-				b.log.Errorf("failed to snapshot data:%s", err)
+				b.log.Panic("failed to snapshot data", logging.Error(err))
 			} else {
 				err = os.Remove(snapshotInProgressFile)
```

Comment:
this err should panic as well to make testing behave, not 100% we should panic in all other circumstances, but should be ok for now

Reply:
addressed.